keyboards/chimera_ortho/chimera_ortho.h

```diff
@@ -63,4 +63,17 @@
 	{ KC_NO,      KC_##k28, KC_##k14, KC_##k00, KC_##k45,      KC_##k48, KC_##k13, KC_##k27, KC_##k44, KC_NO }, \
 }
 
+#define KC_KEYMAP( \
```

Comment:
I'm not sure how I missed this before, but it looks like you have these backwards - the `KC_KEYMAP` should prepend `KC_` where the `KEYMAP` should just pass them straight through :)